lib/roda/plugins/exception_page.rb

```diff
@@ -84,11 +96,20 @@ module RodaPlugins
     # Used under the modified BSD license:
     # http://www.xfree86.org/3.3.6/COPYRIGHT2.html#5
     module ExceptionPage
-      def self.load_dependencies(app)
+      def self.load_dependencies(app, opts=OPTS)
         app.plugin :h
       end
 
-      # Stylesheet used by the HTML exception page
+      # Use the CSS given as the :css option as the exception page's CSS and the
+      # JavaScript given as the :js option as the exception page's JavaScript,
+      # falling back to the defaults when not given.
+      def self.configure(app, opts=OPTS)
+        css, js = opts.fetch(:assets, NOOP).call(+self.css, +self.js)
```

Comment:
I opted for a single configuration point that takes a callable and returns _both_ the CSS and the JS. I waffled on this and thought this would make the most sense since it's uncommon to want this behavior anyway.

Reply:
I'm against the single callable approach.  I understand it makes implementation simpler, but even in the rare cases this is used, only one will be modified (likely the CSS).

I don't think the need to modify the exception page css/js is common enough to warrant plugin options.  As long as we add methods, applications wanting to modify the css/js can override them and call super.